src/layman/tools/rest_tools.py

```diff
@@ -0,0 +1,9 @@
+from flask import Blueprint, jsonify
+
+bp = Blueprint('rest_tools', __name__)
+
+
+@bp.route('/style-info', methods=['GET'])
```

Comment:
Use POST method instead, or add POST to the list
- https://stackoverflow.com/questions/978061/http-get-with-request-body
- https://www.elastic.co/guide/en/elasticsearch/guide/current/_empty_search.html